cmd/scan/scan.go

```diff
@@ -83,6 +83,7 @@ func GetScanCommand(ks meta.IKubescape) *cobra.Command {
 	scanCmd.PersistentFlags().BoolVarP(&scanInfo.Submit, "submit", "", false, "Send the scan results to ARMO management portal where you can see the results in a user-friendly UI, choose your preferred compliance framework, check risk results history and trends, manage exceptions, get remediation recommendations and much more. By default the results are not submitted")
 	scanCmd.PersistentFlags().StringVar(&scanInfo.HostSensorYamlPath, "host-scan-yaml", "", "Override default host scanner DaemonSet. Use this flag cautiously")
 	scanCmd.PersistentFlags().StringVar(&scanInfo.FormatVersion, "format-version", "v1", "Output object can be differnet between versions, this is for maintaining backward and forward compatibility. Supported:'v1'/'v2'")
+	scanCmd.PersistentFlags().StringVar(&scanInfo.CustomClusterName, "clusterName", "", "Set the custom name of the cluster")
```

Comment:
Please change the flag to `cluster-name`.
We should clarify that this flag foes not replace or come instead of the `kube-context` flag

Reply:
> We should clarify that this flag foes not replace or come instead of the kube-context flag

Should I change the flag's usage to : `Set the custom name of the cluster. Not same as the kube-context flag` ?